scripts/test_couler.sh

```diff
@@ -13,6 +13,46 @@
 
 set -e
 
+############# Run Couler unit tests #############
 pip install -r python/couler/requirements.txt
 
 pytest python/couler/tests
+
+
+############# Run Couler e2e test #############
+CHECK_INTERVAL_SECS=2
+cd python/couler/ && python setup.py install
```

Comment:
Default install Couler in the Docker image?

Reply:
Good idea  and I will do that in the next PR.